uni/lib/view/bug_report/widgets/form.dart

```diff
@@ -239,6 +239,7 @@ class BugReportFormState extends State<BugReportForm> {
         : bugDescriptions[_selectedBug]!.item2;
 
     String toastMsg;
+    bool state;
```

Comment:
`state` isn't the right name to be use here. It can confuse people because [state is a design pattern](https://scottt2.github.io/design-patterns-in-dart/state/)
Prefer using `status`